### PR TITLE
Add basic crash reporting to near-cli

### PR DESCRIPTION
**Pull request by janedegtiareva**
This requires adding a workaround to spawn a child process, since
mixpanel does not handle this case properly.

There is an existing limitation that if a user did not opt in to
tracking, the crash report will not be sent. We may want to ask an
additional qustion for this case, since there are scenarios where people
are willing to send an individual crash report but not opt in to general
tracking.

There is a known limitation that this implementation does not log the
environment that was used.

### CODE REVIEW

**Comment by janedegtiareva** [comment]
Tested on windows, ` node bin\near state cad10` produces this in mixpanel 
Screenshot (27)

**Comment by janedegtiareva** [comment]
There is a known issue that if a user fires off 2 commands in a row that result in a crash, only one gets logged. I haven't investigated the root cause. Possible explanations: some sort of event-deduping in mixpanel, process forking.

**Comment by vgrichina** [comment]
Should we integrate with http://getsentry.com to get more context for errors (stack trace, etc)?

**Comment by janedegtiareva** [comment]
> Should we integrate with http://getsentry.com to get more context for errors (stack trace, etc)?

That's a good option for the next iteration @ashleynear @potatodepaulo . I think we'll learn a lot from this one as a easy-to-implement low hanging fruit, so it still makes sense to land this.

**Comment by janedegtiareva** [comment]
This PR was originally based on this POC work by @mikedotexe https://github.com/near/near-shell/commit/e1c5754c10b812217d5815ee5578cef78be4b5a2

File: utils/crash-error-report.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const eventtracking = require('./eventtracking');
+
+const mixPanelProperties = {
+    near_shell_command: process.env.NEAR_CLI_ERROR_LAST_COMMAND,
+    error_message: process.env.NEAR_CLI_LAST_ERROR,
+    network_id: process.env.NEAR_CLI_NETWORK_ID
+};
+Object.assign(mixPanelProperties, {});
```
Comment (mikedotexe) [inline utils/crash-error-report.js:8]: I don't understand this line, isn't this adding nothing to the `mixPanelProperties`?
Looking at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Comment (mikedotexe) [reply utils/crash-error-report.js:8]: tested this by removing this line and it works, I think we can get rid of it

Comment (janedegtiareva) [reply utils/crash-error-report.js:8]: This was a copy paste artifact, thanks for catching!

**Pull request merged**